recipe/meta.yaml

```
@@ -1,52 +1,67 @@
-{% set version = "6.2.1" %}
+{% set name = "gmp" %}
+{% set version = "6.3.0" %}
+{% set posix = 'm2-' if win else '' %}
+{% set am_version = "1.15" %} #  keep synchronized with build.sh
 
 package:
-  name: gmp
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://gmplib.org/download/gmp/gmp-{{ version }}.tar.bz2
-  sha256: eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c
-  patches:
-    - 0000-Avoid-the-x18-register.patch
+  url: https://gmplib.org/download/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: ac28211a7cfb609bae2e2c8d6058d66c8fe96434f740cf6fe2e47b000d1c20cb
 
 build:
-  number: 3
-  skip: true  # [win]
+  number: 0
   run_exports:
     # excellent, even across major ver:
     #  https://abi-laboratory.pro/tracker/timeline/gmp/
     - {{ pin_subpackage("gmp") }}
 
 requirements:
   build:
-    - gnuconfig  # [unix]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - m4
-    - make
-    - patch  # [unix]
-  host:
+    - {{ compiler('c') }}                 # [unix]
+    - {{ compiler('cxx') }}               # [unix]
+    - gnuconfig                           # [unix]
+    - m4                                  # [unix]
+    - make                                # [unix]
+    - m2w64-toolchain                     # [win]
+    # m2-autoconf includes m2-m4 for win
+    - {{posix}}autoconf                   # [win]
+    - {{posix}}automake-wrapper           # [win]
+    - {{posix}}automake{{ am_version }}   # [win]
+    - {{posix}}libtool                    # [win]
+    - posix                               # [win]
```

Future-proof this, please:

```suggestion
    - {{posix}}posix                               # [win]
```

---

Sorry, my bad!  Ignore this comment!